packages/protocol/genesis/GenerateGenesis.g.sol

```diff
@@ -1,33 +1,30 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.20;
 
-import  "lib/openzeppelin-contracts/contracts/proxy/transparent/TransparentUpgradeableProxy.sol";
-import  "forge-std/console2.sol";
-import  "forge-std/StdJson.sol";
-import  "forge-std/Test.sol";
-import  "../contracts/common/AddressManager.sol";
-import  "../contracts/common/AddressResolver.sol";
-import  "../contracts/common/EssentialContract.sol";
-import  "../contracts/bridge/Bridge.sol";
-import  "../contracts/tokenvault/ERC1155Vault.sol";
-import  "../contracts/tokenvault/ERC20Vault.sol";
-import  "../contracts/tokenvault/ERC721Vault.sol";
-import  "../contracts/bridge/IBridge.sol";
-import  "../contracts/test/erc20/RegularERC20.sol";
-import  "../contracts/signal/SignalService.sol";
-import  "../contracts/L2/TaikoL2.sol";
+import "forge-std/console2.sol";
+import "forge-std/StdJson.sol";
+import "forge-std/Test.sol";
+import "../contracts/common/AddressManager.sol";
+import "../contracts/common/AddressResolver.sol";
+import "../contracts/common/EssentialContract.sol";
+import "../contracts/bridge/Bridge.sol";
+import "../contracts/tokenvault/ERC1155Vault.sol";
+import "../contracts/tokenvault/ERC20Vault.sol";
+import "../contracts/tokenvault/ERC721Vault.sol";
+import "../contracts/bridge/IBridge.sol";
+import "../contracts/common/OwnerUUPSUpgradable.sol";
```

Comment:
maybe we can remove imports for 
- OwnerUUPSUpgradable
- AddressManager
- AddressResolver
- IBridge
as they are all indirectly imported by EssentialContract